### PR TITLE
Add space for Ourbloc (Dap)

### PR DESCRIPTION
**Pull request by Ourbloc**
Etherscan: https://etherscan.io/token/0x78a685e0762096ed0f98107212e98f8c35a9d1d8
CoinGecko: https://www.coingecko.com/en/coins/bloc
Website: https://www.daplabsllc.org/
Twitter: https://twitter.com/daplabsllc

### CODE REVIEW

File: spaces/Ourbloc/skin.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.name {
+  --primary-color: #384aff;
+  --bg-color: white;
+  --text-color: #586069;
+  --link-color: #111111;
+  --heading-color: #111111;
+  --border-color: #d1d5da;
+  --header-bg: white;
+  --block-bg: transparent;
+}
```
Comment (e-asia) [inline spaces/Ourbloc/skin.scss:10]: To use the default theme you will want to delete skin.scss from your space folder.

File: spaces/Ourbloc/index.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "key": "Ourbloc",
```
Comment (e-asia) [inline spaces/Ourbloc/index.json:2]: The key will be the url and we want it all in lowercase. Please modify the value of the key by something like `ourbloc` and your space directory too. Thank you.
```suggestion
  "key": "ourbloc",
```

**Review by e-asia — CHANGES_REQUESTED** [review]
Your `space.png` image should be square format.

File: spaces/Ourbloc/skin.scss (hunk 1 above)
Comment (Ourbloc) [reply spaces/Ourbloc/skin.scss]: Thanks a lot for your patience

**Review by e-asia — CHANGES_REQUESTED** [review]
You juste need to rename your space folder from `spaces/Ourbloc/index.json` to `spaces/ourbloc/index.json`. No UPPERCASE are allowed. Thank you.

**Pull request closed without merging**